### DISCUSSION TITLE
Add option to copy/paste applet configuration or mirror panels between several monitors

### DISCUSSION THREAD

**Discussion by suspiciousRaccoon** [linuxmint/discussions#286] [Ideas]
Currently if you want the same layout in several monitors you need to manually configure all of them. Although the customization is great I would prefer if I had to only do it once instead of customizing -> repeat everything in the other monitors.

If I'm not mistaken [this](https://github.com/linuxmint/cinnamon/issues/5352#issuecomment-445454120) feature was present in mint 19, but I wasn't able to find why it was removed. If it was simply moved to somewhere else I wasn't able to find this setting.

**Comment by cerels** [comment]
I came here for the same thing, pretty dure every other DE have this option already

**Comment by OnlineLearningTutorials** [comment]
something similar I suggested for future mint  https://github.com/orgs/linuxmint/discussions/1215

**Comment by Lurux** [comment]
Would be very useful. See also https://github.com/linuxmint/cinnamon/issues/9052
